### ISSUE TITLE
Unison suddenly quits after "reconcile changes" step when displaying changes

### ISSUE THREAD

**Issue by cleterrier** [bcpierce00/unison#266]
Hi,

I have a simple sync job between main hard drive and external drive (~2TB in total size of files), Unison 2.51.2 under OSX 10.14.5.  When trying to sync the scan happens but the last thing Unison displays is "reconciling changes" in the status bar, briefly displays the changes in the main window and then suddenly quits with no crash message. The dump file in my home folder does not contain anything related to this (last entry is from 2018) and the log does not register the Unison run. What can I do? I have tried to delete the ap* fp* files, but this still occurs after rescanning everything.

**Comment by bcpierce00** [comment]
Start by running with "-debug all" or "-debug verbose" on the command line.

Also, try using "-path" to focus on just a small subset of your files.

**Comment by melikyuksel** [comment]
@bcpierce00 How do you even do that when you're using the GUI?

**Comment by bcpierce00** [comment]
You can run Unison from the command line on macOS by doing something like this:

    /Applications/Unison.app/Contents/MacOS/Unison

The debug switch doesn't write to the log; it writes to the standard output stream.  Not sure where that goes if you run from the Finder.

**Comment by melikyuksel** [comment]
@bcpierce00 I just converted the command line flags to the .prf format and added them into the .prf file. I worked on some individual paths, and afterwards, it didn't crash this time while reconciling changes for the entire directory. I turned off permissions, etc. Maybe that's why. But I'm on the macOS version 2.48 (not 2.51) since I'm working with a remote machine that runs Debian which only has version 2.48 of Unison available in its repo.
